pkg/storaged/storage.css

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
+@import "../../node_modules/@patternfly/patternfly/components/Page/page.css";
```

Comment:
OOI, why does that need to happen here? I thought when importing PF-React in overview.jsx it would bundle up the CSS inside the webpack? ISTR that this explicit "include in .css" is only necessary when using `pf-*` classes directly in .html files?

Reply:
There are three components, including this one (Page), which need special handling because they use some PF global variables for breakpoints which we override, see

https://github.com/cockpit-project/cockpit/blob/master/webpack.config.js#L533
and
https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/_global-variables.scss#L8

For these we forbid loading the auto imported CSS files so that we can manually import the SCSS so that our global variables overrides can take effect.

Reply:
Thanks!